cypress/e2e/signIn.cy.js

```diff
@@ -1,11 +1,28 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
-  beforeEach(() => {
+  });
 
+  it('should provide an ability to log in', () => {
+    cy.visit('https://the-internet.herokuapp.com/login');
```

Comment:
```suggestion
    cy.visit('https://the-internet.herokuapp.com/login');
```
move repeated code to beforeEach block